index.js

```diff
@@ -0,0 +1,117 @@
+// 1.
```

Comment:
Please create a file for each question and also upload the questions in a .js file